app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = FoodFacade.food_finder(params[:q])
```

Comment:
What does this return to you? Is it an array with `total_hits` and `foods`?. If so, wondering if you could do:
```
results = FoodFacade.food_finder(params[:q])
@total_hits = results[0]
@foods = results[1]
```
Then in your view you would have access to just call `@total_hits`